Classes/Search/SearchViewController.swift

```diff
@@ -101,12 +102,13 @@ SearchResultSectionControllerDelegate {
     func search(term: String) {
         recentStore.add(recent: term)
 
-        state = .loading
         update(animated: false)
 
-        client.search(query: term, containerWidth: view.bounds.width) { [weak self] resultType in
+        let request = client.search(query: term, containerWidth: view.bounds.width) { [weak self] resultType in
+            guard let state = self?.state, case .loading(_) = state else { return }
```

Comment:
Do you still need the `(_)` here bit? I know in switches, if all parameters are unused you can just not specify them! `case .loading = state`

Reply:
Good catch! 👍